src/ZoneServer/Skills/Handlers/Swordsman/Highlander/CartarStroke.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Melia.Shared.Data.Database;
+using Melia.Shared.Game.Const;
+using Melia.Shared.L10N;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Combat;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.Skills.SplashAreas;
+using Melia.Zone.World.Actors;
+using static Melia.Zone.Skills.SkillUseFunctions;
+
+namespace Melia.Zone.Skills.Handlers.Swordsman.Highlander
+{
+	/// <summary>
+	/// Handler for the Highlander skill Crown.
+	/// </summary>
+	[SkillHandler(SkillId.Highlander_CartarStroke)]
+	public class Highlander_CartarStroke : IGroundSkillHandler, IDynamicCasted
+	{
+		/// <summary>
+		/// Called when the user starts casting the skill.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		public void StartDynamicCast(Skill skill, ICombatEntity caster)
+		{
+			Send.ZC_PLAY_SOUND(caster, "voice_war_atk_long_cast");
+		}
+
+		/// <summary>
+		/// Called when the user stops casting the skill.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		public void EndDynamicCast(Skill skill, ICombatEntity caster)
+		{
+			Send.ZC_STOP_SOUND(caster, "voice_war_atk_long_cast");
+		}
+
+		/// <summary>
+		/// Handles skill, damaging targets.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="originPos"></param>
+		/// <param name="farPos"></param>
+		public void Handle(Skill skill, ICombatEntity caster, Position originPos, Position farPos, ICombatEntity target)
+		{
+			if (!caster.TrySpendSp(skill))
+			{
+				caster.ServerMessage(Localization.Get("Not enough SP."));
+				return;
+			}
+
+			skill.IncreaseOverheat();
+			caster.SetAttackState(true);
+
+			var splashParam = skill.GetSplashParameters(caster, originPos, farPos, length: 45, width: 45, angle: 0);
+			var splashArea = skill.GetSplashArea(SplashType.Circle, splashParam);
+
+			Send.ZC_SKILL_READY(caster, skill, originPos, farPos);
+			Send.ZC_SKILL_MELEE_GROUND(caster, skill, farPos, null);
+
+			this.Attack(skill, caster, splashArea);
+		}
+
+		/// <summary>
+		/// Executes the actual attack after a delay.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="splashArea"></param>
+		private async void Attack(Skill skill, ICombatEntity caster, ISplashArea splashArea)
+		{
+			var hitDelay = TimeSpan.FromMilliseconds(300);
+			var damageDelay = TimeSpan.FromMilliseconds(50);
+			var skillHitDelay = TimeSpan.Zero;
+			var delayBetweenHits = TimeSpan.FromMilliseconds(50);
+
+
+			await Task.Delay(hitDelay);
+
+			List<ICombatEntity> targets;
+			var hits = new List<SkillHitInfo>();
+
+			for (int i = 0; i < 3; i++) { 
```

Comment:
Is this comment about the List<ICombatEntity> targets?  I can't use var there unless I initialize the list, which is unnecessary since it will be replaced later.

Reply:
Nevermind, it seems the PR comment is always about the last line shown, so it's about the for loop.  I made that one var.